tests/baselines/reference/checkIndexConstraintOfJavascriptClassExpression.errors.txt

```diff
@@ -9,7 +9,7 @@ tests/cases/compiler/weird.js(9,17): error TS7006: Parameter 'error' implicitly
     someFunction(function(BaseClass) {
     ~~~~~~~~~~~~
 !!! error TS2552: Cannot find name 'someFunction'. Did you mean 'Function'?
-!!! related TS2728 /.ts/lib.es5.d.ts:321:13: 'Function' is declared here.
+!!! related TS2728 /.ts/lib.es5.d.ts:315:13: 'Function' is declared here.

```

Comment:
I really want to send a PR which changes the baselines to filter out all line numbers from `lib.*.d.ts` files; it really sucks to modify a baseline and have it all be line numbers that don't matter.

Reply:
I was thinking the same - however, maybe start conservative and only do that on related spans.

Reply:
The other problem is that when you add a new overload to a lib, more than just line numbers change; any use gets more symbols too, so it's not a perfect solution.

Reply:
#52330